src/server/lock_manager/waiter_manager.rs

```diff
@@ -110,6 +111,7 @@ pub enum Task {
         lock: Lock,
         timeout: WaitTimeout,
         diag_ctx: DiagnosticContext,
+        wait_timer: Instant,
```

Comment:
If you call this `xx_timer`, what about still using `WAITER_LIFETIME_HISTOGRAM.start_timer()`?

Reply:
According to the existing API definition, it is not convenient to obtain the specific time in the HistogramTimer, so I directly used `WAITER_LIFETIME_HISTOGRAM.Observe (duration_to_sec(elapsed))` to record the metric. I think we could change the name just to make it simple? What about `start_waiting_time` or `enqueue_time` ? What's your opinion?

Reply:
Umm, right.

`start_waiting_time` sounds ok.

Reply:
Fixed~